connect/src/state.rs

```diff
@@ -127,10 +127,15 @@ pub(super) struct ConnectState {
 
     /// a context to keep track of the autoplay context
     autoplay_context: Option<StateContext>,
+
+    // The volume adjustment per step when handling individual volume adjustments.
```

Comment:
Technically not necessary as we don't expose `ConnectState` itself, but would be nice if this could be a doc-comment (just add a `/` at the beginning so that they are three instead of two slash)